spec/features/finance/participant_drilldown_spec.rb

```diff
@@ -38,6 +38,28 @@
       then_i_see(ect_user.id)
       then_i_see(ect_declaration.id)
     end
+
+    scenario "Edit a partipant" do
+      visit finance_participant_path(ect_user)
+
+      actions = page
+                  .find("dt.govuk-summary-list__key", text: "Training status")
+                  .sibling("dd.govuk-summary-list__actions")
+      within(actions) { click_on "Change" }
+
+      select "Withdrawn", from: "Change training status"
+      click_on "Change profile"
+
+      expect(page.find("dt.govuk-summary-list__key", text: "Training status"))
+        .to have_sibling("dd.govuk-summary-list__value", text: "withdrawn")
+
+      actions = page
+                  .find("dt.govuk-summary-list__key", text: "Training status")
+                  .sibling("dd.govuk-summary-list__actions")
+      within(actions) { click_on "Change" }
+
+      expect(page).to have_select("Change training status", selected: "Withdrawn")
+    end
   end
```

Comment:
This is a bit of a change in the style of the feature tests, is it worse to have to abstract everything or to have the inconsistency? 😄